dialogue-target/src/main/java/com/palantir/dialogue/Request.java

```diff
@@ -139,16 +148,21 @@ public static Builder builder() {
 
         private Optional<RequestBody> body = Optional.empty();
 
+        private RequestAttachments attachments;
+
         private int mutableCollectionsBitSet = 0;
 
-        private Builder() {}
+        private Builder() {
+            attachments = DefaultRequestAttachments.create();
```

Comment:
nit: let's keep this null and initialize in `build()`. That way `Builder.from(Request)` doesn't create objects to throw away.

Reply:
ah, misread the code, for some reason I took from as an overloaded constructor.

Reply:
DOne

Reply:
Null analysis was failing, so I had to initialize in the Request constructor.